src/nss-resolve/nss-resolve.c

```diff
@@ -258,6 +250,10 @@ enum nss_status _nss_resolve_gethostbyname4_r(
         *errnop = -r;
```

Comment:
A few lines up:

```c
         /* Explicitly reset all error variables */
         *errnop = 0;
         *h_errnop = NETDB_SUCCESS;
         h_errno = 0;
```

I thought even though we're keeping the reset of h_errno and *h_errnop, we're no longer setting *errnop = 0 anywhere... right?

Reply:
Drop the *errnop reset. You should not be setting errno to 0. Keep the h_errnop/h_errno reset for rhbz 1125975.